webpack.config.js

```diff
@@ -0,0 +1,89 @@
+/** @format */
+/**
+ * External dependencies
+ */
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+const { get } = require( 'lodash' );
+const path = require( 'path' );
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const { DefinePlugin } = require( 'webpack' );
+const WebpackRTLPlugin = require( 'webpack-rtl-plugin' );
+
+/**
+ * WordPress dependencies
+ */
+const CustomTemplatedPathPlugin = require( '@wordpress/custom-templated-path-webpack-plugin' );
+
+const NODE_ENV = process.env.NODE_ENV || 'development';
+
+const externals = {
+  '@wordpress/api-fetch': 'window.wp.apiFetch',
+  '@wordpress/blocks': { this: [ 'wp', 'blocks' ] },
+  '@wordpress/components': { this: [ 'wp', 'components' ] },
+  '@wordpress/compose': { this: [ 'wp', 'compose' ] },
+  '@wordpress/data': { this: [ 'wp', 'data' ] },
+  '@wordpress/editor': { this: [ 'wp', 'editor' ] },
+  '@wordpress/element': 'window.wp.element',
+  '@wordpress/hooks': 'window.wp.hooks',
+  '@wordpress/html-entities': { this: [ 'wp', 'htmlEntities' ] },
+  '@wordpress/i18n': 'window.wp.i18n',
+  '@wordpress/keycodes': { this: [ 'wp', 'keycodes' ] },
+  '@woocommerce/components': 'window.wc.components',
+  tinymce: 'tinymce',
+  moment: 'moment',
+  react: 'React',
+  'react-dom': 'ReactDOM',
```

Comment:
This appears to be something related to `lodash` being already brought in. I'd have to dig through some of the @wordpress babel presets and configs to find out for sure whats happening, but since `lodash` is already on the window, you can add this line to the `externals` object:

```js
lodash: 'lodash',
```
Not only does this work, but it reduces the bundle size by relying the `lodash` brought in by Gutenberg. Why wc-admin works without this line, I don't know, but we should also add it in that repo.

Just to double check, I added a dependency at random:

```
npm install --save reverse-string
```
and in as-report.js:

```js
import reverse from 'reverse-string';
console.log( reverse( 'test' ) );
```
and its all good.

Reply:
@psealock Thanks, that worked for me as well. @timmyc Your example is broken for the same reason.